lib/hound/session_server.ex

```diff
@@ -22,6 +22,17 @@ defmodule Hound.SessionServer do
     end
   end
 
+  def current_session_name(pid) do
+    case :ets.lookup(@name, pid) do
+      [{^pid, _ref, session_id, all_sessions}] ->
+        Enum.find_value all_sessions, fn
+          {name, ^session_id} -> name
```

Comment:
It seems this fails to compile in travis (`** (CompileError) lib/hound/session_server.ex:29: unbound variable ^session_id`). I had no problems compiling or testing locally (Elixir 1.2.4) is this something that has changed since 1.0.4?